app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_ = {}
+
+    def inner(*args) -> Any:
+        key = args
```

Comment:
Why is this variable needed? Just use `args` instead of creating varible with the ssame value